Add snowflake session, schemata, login_history and account_parameter table; Add common columns `account` and `region` to all tables; Add `warehouse` connection config argument (#2)
turbot/steampipe-plugin-snowflake: docs/tables/snowflake_session.md

# Table: snowflake_session

This Account Usage view provides information on the session, including information on the authentication method to Snowflake and the Snowflake login event. Snowflake returns one row for each session created over the last year.

> **Note**:

Reviewer: ```suggestion
**Notes**:
```

- This table requires a `Snowflake warehouse` to query. You can set it by `warehouse` config argument in Steampipe connection config.
- Latency for the view may be up to 180 minutes (3 hours).

## Examples

### Basic info

```sql
select
  session_id,
  user_name,
  authentication_method,
  created_on,

>   client_environment ->> 'APPLICATION' as clint_application

Reviewer: ```suggestion
  client_environment ->> 'APPLICATION' as client_application
```

Please fix any typos before pushing up for review

from
  snowflake_session;
```

### List distinct authentication methods used in last year

```sql
select distinct
  user_name,
  authentication_method
from
  snowflake_session
order by
  user_name;
```

> ### List sessions authencticated without Snowflake MFA with passsword in 30 days

Reviewer: ```suggestion
### List sessions authenticated without Snowflake MFA with password in last 30 days
```

Please be careful of typos (multiple in this title), use a spellchecker if required
